src/components/ChatApp/ChatApp.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect } from 'react';
+import Popup from '../Popup';
+import SubjectList from '../SubjectList';
+import * as api from '../../api';
+import Chat from '../Chat';
+import LoadingIndicator from '../LoadingIndicator';
+import { useCookie } from '../../useCookie';
+//import ErrorBoundary from './ErrorBoundary';
```

Comment:
It's not really important, but having commented code can make files cluttered fast. So I usually avoid having them commited to the repositories but you can decide what you want to do with this.

Reply:
removed